From 3f92137286dcb565a70337620a1fb94b59a79c89 Mon Sep 17 00:00:00 2001
From: Subbaraya Sundeep Bhatta <subbaraya.sundeep.bhatta@xilinx.com>
Date: Mon, 23 Sep 2013 21:20:40 +0530
Subject: [PATCH 550/628] usb: gadget: Remove platform_set_drvdata setup to
 NULL

git://github.com/Xilinx/linux-xlnx.git xilinx-v14.7
commit ddc4b0e16486f377e825d19cc24f9bbd843e87cf

It is done automatically.

Signed-off-by: Subbaraya Sundeep Bhatta <sbhatta@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 drivers/usb/gadget/xilinx_udc.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/usb/gadget/xilinx_udc.c b/drivers/usb/gadget/xilinx_udc.c
index 7fe939b..71baf6a 100644
--- a/drivers/usb/gadget/xilinx_udc.c
+++ b/drivers/usb/gadget/xilinx_udc.c
@@ -2161,7 +2161,6 @@ static int xudc_remove(struct platform_device *pdev)
 	if (udc->driver)
 		return -EBUSY;
 
-	platform_set_drvdata(pdev, NULL);
 	device_unregister(&udc->gadget.dev);
 
 	return 0;
-- 
1.7.5.4

